deploy/infrastructure/examples/README.md

```diff
@@ -0,0 +1,68 @@
+# DSS Infrastructure Examples
```

Comment:
Wouldn't it be better to have this file at the top level in `/deploy/README.md`? I get that it's specific to the examples folder, but it could make sens that this is the first thing that comes up to someone that just opened the project for the first time

Reply:
I am going step by step. I will create the /deploy/README.md in an upcoming PR when appropriate.